media/css/products/vpn/components/block.scss

```diff
@@ -74,6 +74,36 @@ $image-path: '/media/protocol/img';
         }
     }
 
+    &.platform-get-started-cta {
```

Comment:
If there's a better naming convention, please advise!

Reply:
I'd say let's use something that describes the layout that the modifier creates, rather than something that ties it to the platform pages?